wp-content/themes/inf_theme/theme/class-theme.php

```diff
@@ -58,22 +36,22 @@ public function enqueue_styles() {
    * 'enqueued' with empty string. This is done to avoid multiple jQuery loading, since
    * one is bundled with webpack and exposed to the global window.
    *
-   * @since 2.0.0
+   * @since 1.0.0
    */
   public function enqueue_scripts() {
     // jQuery.
     wp_deregister_script( 'jquery-migrate' );
     wp_deregister_script( 'jquery' );
-    wp_register_script( 'jquery', get_template_directory_uri() . '/skin/public/scripts/vendors/jquery.min.js', array(), '3.3.1' );
+    wp_register_script( 'jquery', get_template_directory_uri() . '/skin/public/scripts/vendors/jquery.min.js', array(), '3.3.1' ); // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.NotInFooter
```

Comment:
Likewise about not being in the footer.

Reply:
in all WP themes, Jquery is never loaded in the footer so we shouldn't neither.

Reply:
Just because they do it, doesn't mean it7s the right way to do it 😄 They are still using bundled jQuery (1.2) xD

I think we should put our scripts in footer, and make them dependent on the bundled vendor jquery. This way we are not blocking down the HTML parsing on the page (which will happen if the script is in the header).

If we put them in the header, we should investigate in adding `async` and `defer` attributes on our scripts.

[Reference](https://developers.google.com/speed/docs/insights/BlockingJS)

Reply:
I agree with you 100%, but we can't guarantee that plugin developers will hook their scripts to the footer or dependent on jQuery, and if the script is loaded in header and jQuery is in the footer it won't work.

Reply:
How about adding it as an option? Radio button with "Load in header, load in header async, load in footer?"

Reply:
Not a bad idea, but I think that it would be better to leave that to the users, and just present the minimal working version of the developers to use and enhance :)